src/bluehawk/actions/snip.ts

```diff
@@ -69,9 +78,11 @@ export const formatInRst = async (
     for (const range of emphasizeAttributes.ranges) {
       const start = await document.getNewLocationFor(range.start);
       const end = await document.getNewLocationFor(range.end);
-      if (start !== undefined && end !== undefined) {
-        rstEmphasizeRanges.push({ start: start.line, end: end.line });
-      }
+      rstEmphasizeRanges.push({
+        // ternary operator here because start or end are both optional, but one of them must be defined
+        start: start ? start.line : end!.line,
```

Comment:
What if `start` is 0?

Reply:
Good call, this is one of those javascript things that *always* gets me. Thanks!